fastlane_core/lib/fastlane_core/clipboard.rb

```diff
@@ -0,0 +1,20 @@
+require 'fastlane_core'
+require 'open3'
+
+module FastlaneCore
+  class Clipboard
+    def self.copy(content: nil)
+      return UI.crash!("'pbcopy' or 'pbpaste' command not found.") unless is_supported?
+      Open3.popen3('pbcopy') { |input, _, _| input << content }
```

Comment:
I wonder if it is worth to make a macOS and Linux version of this, taking advantage of the refactor, just as a suggestion ofc